ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -37,9 +37,7 @@ typedef NS_ENUM(NSInteger, DeviceType) {
 @import CoreTelephony;
 #endif
 
-#if !TARGET_OS_TV
```

Comment:
Why was this removed?

Reply:
Due to the introduction of [getStartupTime](https://github.com/react-native-device-info/react-native-device-info/blob/master/ios/RNDeviceInfo/RNDeviceInfo.m#L1062) and the usage of `kinfo_proc`. Now tvos also requires this import. I updated the PR description.